### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by y-lohse**
See commit titles for details.

### CODE REVIEW

File: packages/cozy-procedures/src/components/documents/DocumentHolder.jsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ class DocumentHolder extends Component {
             }
           />
         )}
-        <Chip variant="dashed" className="u-w-100">
+        <Card className="u-flex u-flex-row u-flex-items-center">
```
Comment (ptbrowne) [inline packages/cozy-procedures/src/components/documents/DocumentHolder.jsx]: So simple :)

File: packages/cozy-procedures/src/components/documents/LoadingDocumentHolder.jsx [hunk 2]
```diff
@@ -1,13 +1,13 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { translate, Chip, Icon, Spinner } from 'cozy-ui/transpiled/react/'
+import { translate, Card, Icon, Spinner } from 'cozy-ui/transpiled/react/'
 
 const LoadingDocumentHolder = ({ t }) => (
-  <Chip variant="dashed" className="u-w-100">
+  <Card className="u-flex u-flex-row u-flex-items-center">
```
Comment (ptbrowne) [inline packages/cozy-procedures/src/components/documents/LoadingDocumentHolder.jsx]: Ouh, second time, should TextCard be used in these cases ?

File: packages/cozy-procedures/package.json [hunk 3]
```diff
@@ -12,6 +12,7 @@
   },
   "dependencies": {
     "babel-preset-cozy-app": "1.5.1",
+    "cozy-doctypes": "^1.49.3",
```
Comment (Crash--) [inline packages/cozy-procedures/package.json]: Why not. I understand the why, but to be coherent we should put it as peerDep ? :d

Comment (y-lohse) [reply packages/cozy-procedures/package.json]: Damnit yes, I told it --peer.

File: packages/cozy-procedures/src/components/documents/DocumentHolder.jsx (hunk 1 above)
Comment (y-lohse) [reply packages/cozy-procedures/src/components/documents/DocumentHolder.jsx]: I guess it's subjective but I really don't find the declaration that complicated. It's not like they were vastly different css rules, they are all flexbox-related and the names are pretty explicit.

**Pull request merged**